src/Chapter1.hs

```diff
@@ -653,8 +675,7 @@ You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
 
-firstDigit n = error "firstDigit: Not implemented!"
-
+firstDigit n = head $ digToList n
```

Comment:
That's very creative solution 🎨  I like it!

In the next Chapter we explore lists in Haskell more, so there will be more input on this and in particular usage of `++` for the solution and how it coiuld affect the performance 👌🏼